browser/mocha/fixture.js

```diff
@@ -14,6 +14,10 @@ extendInterfaces('fixture', function(context, teardown) {
 
     // Find the test-fixture with the provided ID and create it, returning
     // the results:
-    return document.getElementById(fixtureId).create(model);
+    var fixture = document.getElementById(fixtureId);
+    if (!fixture) {
+      throw 'Could not find fixture with ID <' + fixtureId + '>'; 
```

Comment:
Also, can you move this check up to the top of the function body, so that teardown isn't setup with a bad ID?